video/out/opengl/context_wayland.c

```diff
@@ -77,8 +78,13 @@ static void wayland_egl_swap_buffers(struct ra_ctx *ctx)
     struct priv *p = ctx->priv;
     struct vo_wayland_state *wl = ctx->vo->wl;
 
+    int disable_vsync;
+    mp_read_option_raw(ctx->global, "wayland-disable-vsync", &m_option_type_flag,
+                       &disable_vsync);
```

Comment:
To be honest I think each addition of such a call is a bad thing.
Can#t this be grouped with other wayland options? Or if there are none yet, make a new option struct.